<?xml version="1.0"?>
<launch>
    <arg name="use_sim_time" default="false"/>
    <arg name="robot_name" default="rUBot_mecanum"/>
    <arg name="serial_port" default="/dev/ttyACM0"/>
    <arg name="baud_rate" default="57600"/>
    <arg name="loop_rate" default="30"/>
    <arg name="encoder_cpr" default="2550"/>

    <node pkg="my_robot_driver" executable="rubot_nano_driver_mecanum_exec" name="rubot_nano_driver_mecanum" output="screen">
        <param name="robot_name" value="$(arg robot_name)"/>
        <param name="serial_port" value="$(arg serial_port)"/>
        <param name="baud_rate" value="$(arg baud_rate)"/>
        <param name="loop_rate" value="$(arg loop_rate)"/>
        <param name="encoder_cpr" value="$(arg encoder_cpr)"/>
        <param name="use_sim_time" value="$(arg use_sim_time)"/>
    </node>
</launch>